index.html

```diff
@@ -370,14 +372,117 @@ <h1 id="contact-me">Contact me</h1>
                   maxlength="500"
                   placeholder="write your message here..."
                   required
+                  data-field="taxts"
                 ></textarea>
               </li>
             </ul>
+            <span class="error" aria-live="polite"></span>
             <button type="submit" class="first-button">Get In Touch</button>
```

Comment:
Great job team 🤝

- [ ] [Optional] You may kindly fix your submit button correctly. It doesn't appear well centered as the screen size increases. Try to make it a closer replica of the design provided in the Figma template.
![button](https://user-images.githubusercontent.com/63917725/180593688-0472bba3-9b6a-4a67-a1b3-f60c8e985072.PNG)